app/views/orders/seller_items.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Order Number <%= order_id %></h1>
+<table class="table table-striped" class="table table-hover">
+  <thead>
+    <tr>
+      <th>Item</th>
+      <th>Quantity</th>
+      <th>Item Price</th>
+      <th>Item Total</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% order_items = Orderitem.where(order_id: @order.id) && Orderitem.where(seller_id: user_id) %>
+    <% orders = order_items.collect(&:order_id).uniq %>
+    <% orders.each do |order| %>
+      <% orders = Orderitem.where(order_id: order.id) %>
+      <tr>
+      <% orders.each do |order| %>
+        <td><%= order.product.name %></td>
+        <td><%= order.quantity %></td>
+        <td>$<%= order.price %>0</td>
+        <td>$<%= order.price * order.quantity %>0</td>
```

Comment:
Why is there a 0 here? Maybe check out some rails methods that convert pennies into dollars.